### ISSUE TITLE
v5 docs: missing (clear) visible focus indication for version switch button and code samples

### ISSUE THREAD

**Issue by patrickhlauke** [twbs/bootstrap#36506]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

In the current https://getbootstrap.com/ site, the version switch button in the header has a minimal focus indication (just the text colour getting a bit lighter).

Further down in the page, any `<pre>` code sample is focusable, but focus indication has been suppressed.

https://user-images.githubusercontent.com/895831/172058438-ee48720d-b080-4dce-b671-f8ab70ad288a.mp4

### Reduced test cases

https://getbootstrap.com/

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Bootstrap are you using?

5.2

**Comment by patrickhlauke** [comment]
for the `<pre>` elements, the problem is localised in `docs.scss`

```
.highlight pre:focus {
    outline: 0;
}
```

---

for the dropdown button, the issue is more fundamental: in `buttons.scss` the outline is suppressed in favour of the coloured border.

```
.btn-check:focus+.btn, .btn:focus {
    color: var(--bs-btn-hover-color);
    background-color: var(--bs-btn-hover-bg);
    border-color: var(--bs-btn-hover-border-color);
    outline: 0;
    box-shadow: var(--bs-btn-focus-box-shadow);
}
```

I would actually be tempted to suppress the `outline` only in case where `.btn-check:focus:not(:focus-visible)+.btn, btn:focus:not(:focus-visible)` (but keep the other styles there for the more general `:focus` case

/cc @mdo

**Comment by patrickhlauke** [comment]
Going to try out how this would look here https://github.com/twbs/bootstrap/pull/36507
